index.js

```diff
@@ -50,7 +62,25 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies) {
+  // let targetExist = false;
+  // let i = 0;
+
+  // while(i<array.length){
+  //   if (array[i] === selected){
+  //     targetExist = true;
+  // } i++;
+  // }
+  // return targetExist
+  
+  if(!movies.length){
+    throw "error message"
+  }
+  let newArr = movies.some((movie) => {
+    movie.rated ? true : false
```

Comment:
```suggestion
   return movie.rated === rating ? true : false
```
Good use of ternary operator. Return keyword was missing. Need to compare the movie's rating to the inputted `rating`